src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![doc = include_str!("../README.md")]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![warn(missing_docs)]
```

Comment:
I suggest using `[lints]` in `Cargo.toml`

Reply:
Makes sense.